java/core/src/main/java/com/microsoft/bond/StructBondType.java

```diff
@@ -344,6 +358,16 @@ protected final TStruct deserializeValue(TaggedDeserializationContext context) t
         return value;
     }
 
+    @Override
+    protected final TStruct deserializeValue(UntaggedDeserializationContext context) throws IOException {
+        TStruct value = this.newDefaultValue();
+        if (this.baseStructType != null) {
+            this.baseStructType.deserializeValueAsBase(context, value);
+        }
+        this.deserializeStructFields(context, value);
```

Comment:
Observe that due to absence of readStructBegin and readStructEnd, the logic to deserialize a struct and a base is the same. This method can just call deserializeValueAsBase (code is identical).